blint/lib/utils.py

```diff
@@ -207,6 +216,37 @@ def is_ignored_file(file_name):
     return any(file_name.endswith(ie) for ie in ignore_files)
 
 
+def blintdb_setup(args):
+    """
+    This function downloads blint-db package from 'ghcr.io/appthreat/blintdb-vcpkg' using oras client
+    and puts it into $BLINTDB_LOC path.
+    If there is not path in $BLINTDB_LOC, it will add it to $HOME/blindb.
+    $USE_BLINTDB is required to be set "true" or "1", in order to use blintdb
+    """
+    if not os.getenv("USE_BLINTDB") or not args.use_blintdb :
+        LOG.debug(f"skipping blintdb setup {os.getenv('USE_BLINTDB')}")
+        return
+
+    if args.blintdb_home:
+        blintdb_home = args.blintdb_home
+    else:
+        LOG.debug("Creating BLINTDB_LOC os environment")
+        blintdb_home = os.path.join(os.getenv("HOME"), "blintdb")
+        os.environ['BLINTDB_LOC'] = os.path.join(blintdb_home, "blint.db")
```

Comment:
@aryan-rajoria Let's avoid trying to set permanent environment variables. Move `BLINTDB_HOME` and `BLINT_LOC` to the config file, then import into modules where you need them. That way, we will never end up assuming these variables are set and error out by operating on `None` in your db.py functions. That does mean we will define them whether or not blint_db is used, but that's ok.